src/snowflake/cli/plugins/sql/commands.py

```diff
@@ -24,6 +24,7 @@
     variables_option,
 )
 from snowflake.cli.api.commands.snow_typer import SnowTyperFactory
+from snowflake.cli.api.exceptions import IncompatibleParametersError
```

Comment:
Could you add it to other commands as well? I found that `snow object create` has object arguments which are exclusive with `--json object_definition` flag and `snow app events` have conflicting `--first` and `--last`